### ISSUE TITLE
rsmtool hangs when generating the report

### ISSUE THREAD

**Issue by aoifecahill** [EducationalTestingService/rsmtool#97]
When generating multiple reports sequentially (automatically), rsmtool seems to hang at the report generation stage: `INFO:traitlets:Executing notebook with kernel: python3`. 

There are no other rsmtool or ipython processes running on the machine at the time. 

I tried adding in a sleep command after each report (for 1 minute), but that did not help. 

Unfortunately, it's not reproducible. If I run the same sequence multiple times, sometimes it hangs, sometimes it doesn't. And when it does hang it's not necessarily at the same point.

**Comment by aoifecahill** [comment]
Turning the logging level to debug it hung here: 

```
# There's a weird deprecation warning that comes from within statsmodels
# code which is used by seaborn to compute the kernel density estimate.
# Until statsmodels fixes it, we need to ignore that warning.
with warnings.catch_warnings():
    warnings.filterwarnings('ignore', category=np.VisibleDeprecationWarning)
    with sns.axes_style('white'):
        g = sns.FacetGrid(col='variable', data=df_train_preproc_melted, col_wrap=3, 
                          col_order=column_order, sharex=False, sharey=False, size=2, 
                          aspect=1)
        g.map(sns.distplot, "value", color="grey")
        for ax, cname in zip(g.axes, g.col_names):
            labels = ax.get_xticks()
            ax.set_xlabel('')
            ax.set_xticklabels(labels,rotation=90)
            ax.set_title(cname)
        plt.tight_layout(h_pad=1.0)
        plt.savefig(join(figure_dir, '{}_distrib.svg'.format(experiment_id)))
        plt.show()
```

This happened on two occasions that I saw. It could also be hanging in different places.

**Comment by aoifecahill** [comment]
Update: if I add `kde=False` to the `distplot()` call in `preprocessed_features` notebook, I do not see the hanging.

**Comment by desilinguist** [comment]
This should be fixed once we add thumbnails in #113.

**Comment by jbiggsets** [comment]
This should be resolved in the refactored branch. The `kde` argument is being set to `False`.
